scripts/ci-is-generated-in-sync.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
Can we move this to `hack/verify-generated-files.sh`? All the other generated scripts are in there as well.

We'll also need to add the script to the `make verify` target

Reply:
I thought we had that. I'll check.